pkg/utils/printer/printer.go

```diff
@@ -209,6 +255,10 @@ type DefaultMsgPrint struct {
 	Message interface{} `json:"Message,omitempty"`
 }
 
+type ToPrint struct {
+	Message string
+}
+
```

Comment:
Why do we need to encapsulate this string here?

Reply:
there is no need, first i wanted to create a more complex print structure, but I ended using just a message, i removed it and now it's creating the json using just the string